docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -236,6 +236,8 @@ The following configuration set `Content-Type: application/vnd.android.package-a
 - `CUSTOM_URL_SCHEMES`: Use a comma separated list (ftp,git,svn) to indicate additional
   URL hyperlinks to be rendered in Markdown. URLs beginning in http and https are
   always displayed
+- `ENABLE_MATH`: **true**: Enables detection of `\(...\)`, `\[...\]` and `$$...$$` blocks as math blocks.
+- `ENABLE_INLINE_DOLLAR_MATH`: **true**: In addition enables detection of `$...$` as inline math.
```

Comment:
Hmm, I'm still not sure:
Why isn't this included in `ENABLE_MATH`?
Especially as it is probably the most well-known form of starting math content...
I don't really see the reason for handling this separately.
I've looked through the previous conversation on this PR, and the only thing I could find regarding this was `KaTeX strongly discourages it` as it could lead to parse irregularities.
However, I looked through their documentation, and couldn't find anything regarding that.
And as it is at the moment, it seems pretty arbitrary why only the inline dollar math has its own setting.

Reply:
I've just tried to find the document I've read it in but it's really not hard to see why allowing `$` inline math could be 
very annoying:

The current average price of regular gasoline in California $5.394 / gallon – yesterday it was $5.374

Reply:
done